### PR TITLE
added coming soon flag to sp section

### PR DESCRIPTION
**Pull request by ChrisArnoldMSFT**

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Teams/teams-learning-app-overview.md [hunk 1]
```diff
@@ -108,7 +108,7 @@ Other sources might need to be enabled or configured manually. Learning sources
 
 To enable or disable a learning content source, select the check box next to the source. If a source is enabled, a check mark will be visible.
 
-## Configure SharePoint as a learning content source
+## Configure SharePoint as a learning content source (Coming Soon)
```
Comment (JohanFreelancer9) [inline Teams/teams-learning-app-overview.md:111]:
```suggestion
## Configure SharePoint as a learning content source (coming soon)
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

File: Teams/teams-learning-app-overview.md (hunk 1 above)
Comment (ChrisArnoldMSFT) [reply Teams/teams-learning-app-overview.md:111]: @JohanFreelancer9 @scanum thank you for the review! How long do the PR merge typically take (when can we expect to see it reflected in the browser)?

**Pull request merged**